dart/lib/src/default_integrations.dart

```diff
@@ -7,16 +7,18 @@ import 'sentry.dart';
 import 'sentry_options.dart';
 import 'throwable_mechanism.dart';
 
-/// integration that capture errors on the runZonedGuarded error handler
+/// Integration that runs runner function within runZonedGuarded
```

Comment:
I believe the new docs should merge the previous docs + running the runner.
This integration still captures errors from the runZonedGuarded zone, right?